### PR TITLE
New accounts for data-platform

### PR DESCRIPTION
**Pull request by SteveLinden**
Additional environments for test, preproduction and production

### CODE REVIEW

File: environments/data-platform.json [hunk 1]
```diff
@@ -33,6 +33,45 @@
...
+    {
+      "name": "test",
+      "access": [
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
+        },
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
```
Comment (markgov) [inline environments/data-platform.json:46]: you have two view only's should one of them not be developer?

File: environments/data-platform.json [hunk 2]
```diff
@@ -33,6 +33,45 @@
...
+    {
+      "name": "preproduction",
+      "access": [
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
+        },
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
```
Comment (markgov) [inline environments/data-platform.json:59]: you have two view only's should one of them not be developer?

File: environments/data-platform.json [hunk 3]
```diff
@@ -33,6 +33,45 @@
...
+    {
+      "name": "production",
+      "access": [
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
+        },
+        {
+          "github_slug": "data-platform-labs",
+          "level": "view-only"
```
Comment (markgov) [inline environments/data-platform.json:72]: you have two view only's should one of them not be developer?

File: environments/data-platform.json (hunk 1 above)
Comment (SteveLinden) [reply environments/data-platform.json:46]: Corrected

File: environments/data-platform.json (hunk 2 above)
Comment (SteveLinden) [reply environments/data-platform.json:59]: Corrected

File: environments/data-platform.json (hunk 3 above)
Comment (SteveLinden) [reply environments/data-platform.json:72]: Corrected

**Review by markgov — APPROVED** [review]
LGTM

**Pull request merged**